packages/ui-components/src/components/Modal/Modal.test.tsx

```diff
@@ -297,8 +297,10 @@ describe("Modal", () => {
     )
     expect(screen.getByRole("dialog")).toBeInTheDocument()
     expect(screen.getByRole("textbox")).toBeInTheDocument()
-    waitFor(() => {
-      expect(screen.getByRole("textbox")).toHaveFocus()
+
+    const _p = waitFor(() => {
```

Comment:
We can remove this variable it works without it

Reply:
It complains about floating promise - I added lint ignore rule.